src/ext-test/java/org/opentripplanner/ext/legacygraphqlapi/GraphQLIntegrationTest.java

```diff
@@ -0,0 +1,196 @@
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.time.Instant;
+import java.time.OffsetDateTime;
+import java.time.temporal.ChronoUnit;
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Locale;
+import javax.annotation.Nonnull;
+import org.glassfish.jersey.message.internal.OutboundJaxrsResponse;
+import org.junit.jupiter.api.parallel.Execution;
+import org.junit.jupiter.api.parallel.ExecutionMode;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.opentripplanner._support.time.ZoneIds;
+import org.opentripplanner.ext.fares.impl.DefaultFareService;
+import org.opentripplanner.framework.i18n.NonLocalizedString;
+import org.opentripplanner.model.plan.Itinerary;
+import org.opentripplanner.routing.alertpatch.AlertCause;
+import org.opentripplanner.routing.alertpatch.AlertEffect;
+import org.opentripplanner.routing.alertpatch.AlertSeverity;
+import org.opentripplanner.routing.alertpatch.EntitySelector;
+import org.opentripplanner.routing.alertpatch.TimePeriod;
+import org.opentripplanner.routing.alertpatch.TransitAlert;
+import org.opentripplanner.routing.api.request.RouteRequest;
+import org.opentripplanner.routing.graph.Graph;
+import org.opentripplanner.routing.graphfinder.GraphFinder;
+import org.opentripplanner.routing.vehicle_parking.VehicleParking;
+import org.opentripplanner.routing.vehicle_rental.VehicleRentalService;
+import org.opentripplanner.service.vehiclepositions.internal.DefaultVehiclePositionService;
+import org.opentripplanner.standalone.config.framework.json.JsonSupport;
+import org.opentripplanner.standalone.server.TestRoutingService;
+import org.opentripplanner.test.support.FilePatternSource;
+import org.opentripplanner.transit.model._data.TransitModelForTest;
+import org.opentripplanner.transit.model.basic.TransitMode;
+import org.opentripplanner.transit.model.framework.Deduplicator;
+import org.opentripplanner.transit.service.DefaultTransitService;
+import org.opentripplanner.transit.service.StopModel;
+import org.opentripplanner.transit.service.TransitModel;
+
+@Execution(ExecutionMode.CONCURRENT)
+class GraphQLIntegrationTest {
+
+  static final Graph graph = new Graph();
+
+  static final Instant ALERT_START_TIME = OffsetDateTime
+    .parse("2023-02-15T12:03:28+01:00")
+    .toInstant();
+  static final Instant ALERT_END_TIME = ALERT_START_TIME.plus(1, ChronoUnit.DAYS);
+
+  private static final LegacyGraphQLRequestContext context;
+
+  static {
+    graph
+      .getVehicleParkingService()
+      .updateVehicleParking(
+        List.of(
+          VehicleParking
+            .builder()
+            .id(id("parking-1"))
+            .name(NonLocalizedString.ofNullable("parking"))
+            .build()
+        ),
+        List.of()
+      );
+
+    var stop = TransitModelForTest.stop("stop1").build();
+    var stopModel = StopModel.of().withRegularStop(stop).build();
```

Comment:
Can you add the stops from the itinerary to the stop model? They are accessible as `A.stop`, `B.stop`, etc